pkg/flink/internal/results/result_formatter_test.go

```diff
@@ -383,3 +381,53 @@ func (s *ResultFormatterTestSuite) TestTruncateMultiLineStringShouldNotTruncate(
 		require.Equal(s.T(), testCase.expected, TruncateString(testCase.input, testCase.maxCharCountToDisplay))
 	}
 }
+
+func (s *ResultFormatterTestSuite) TestTruncateMultibyteCharacters() {
+	testCases := []struct {
+		input                 string
+		expected              string
+		maxCharCountToDisplay int
+	}{
+		{
+			input:                 "あああ",  // The string width here is 6 since each character is 2 bytes wide.
+			expected:              "あ...", // あ... is exactly 5 bytes wide.
+			maxCharCountToDisplay: 5,
+		},
+		{
+			input:                 "SELECT あ FROM ",
+			expected:              "SELECT あ FROM ",
+			maxCharCountToDisplay: 15,
+		},
+		{
+			input:                 "SELECT あ FROM \n table",
+			expected:              "SELECT あ FROM \n table",
```

Comment:
nit:
I understand that the main purpose is just to test line cutting here
however how about to have a more valid SQL to avoid any confusions in future like 
```suggestion
			input:                 "SELECT 'あ' FROM \n table",
			expected:              "SELECT 'あ' FROM \n table",
```
?

Reply:
Sure, I'll update it

Reply:
You probably mean backticks, right?

Reply:
single quotes for string values, 
backticks for catalog, db, table, column names

Reply:
Okay, in this case I meant あ as a column name, so updated tests with backticks